app/lib/features/settings/widgets/settings_menu.dart

```diff
@@ -15,45 +17,149 @@ class SettingsMenu extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final currentRoute = ref.watch(currentRoutingLocation);
     final size = MediaQuery.of(context).size;
-    bool isSelected(Routes route) {
-      debugPrint(
-        '${route.route} $currentRoute, ${currentRoute == route.route}',
+    Color? colorSelected(Routes route) => currentRoute == route.route
+        ? AppTheme.brandColorScheme.secondary
+        : null;
+
+    TextStyle titleStylesSelected(Routes route) {
+      return TextStyle(
+        color: colorSelected(route),
       );
-      return currentRoute == route.route;
     }
 
+    final shouldGoNotNamed = isDesktop || size.width > 770;
+
     return Scaffold(
       appBar: AppBar(
         title: Text(
-          'App Settings',
+          'Settings',
           style: Theme.of(context).textTheme.titleLarge,
         ),
         centerTitle: false,
       ),
-      body: Column(
-        mainAxisAlignment: MainAxisAlignment.start,
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          ListTile(
-            title: const Text('Labs'),
-            subtitle: const Text('Experimental Acter features'),
-            leading: const Icon(Atlas.lab_appliance_thin),
-            selected: isSelected(Routes.settingsLabs),
-            onTap: () {
-              isDesktop || size.width > 770
-                  ? context.goNamed(Routes.settingsLabs.name)
-                  : context.pushNamed(Routes.settingsLabs.name);
-            },
+      body: SettingsList(
```

Comment:
I wonder if we can use this component for room profile. But we should consider that when focusing on room/chat profiles.